tests/unit/Application/Metadata/JsonGeneratorTest.php

```diff
@@ -198,6 +198,24 @@ public function test_it_can_build_saml_entity_data_for_new_entities()
                             'sp' => 'sp',
                             'AssertionConsumerService:0:Binding' => 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST',
                             'AssertionConsumerService:0:Location' => 'http://acs',
+                            'AssertionConsumerService:1:Binding' => null,
+                            'AssertionConsumerService:1:Location' => null,
+                            'AssertionConsumerService:2:Binding' => null,
+                            'AssertionConsumerService:2:Location' => null,
+                            'AssertionConsumerService:3:Binding' => null,
+                            'AssertionConsumerService:3:Location' => null,
+                            'AssertionConsumerService:4:Binding' => null,
+                            'AssertionConsumerService:4:Location' => null,
+                            'AssertionConsumerService:5:Binding' => null,
+                            'AssertionConsumerService:5:Location' => null,
+                            'AssertionConsumerService:6:Binding' => null,
+                            'AssertionConsumerService:6:Location' => null,
+                            'AssertionConsumerService:7:Binding' => null,
+                            'AssertionConsumerService:7:Location' => null,
+                            'AssertionConsumerService:8:Binding' => null,
+                            'AssertionConsumerService:8:Location' => null,
+                            'AssertionConsumerService:9:Binding' => null,
+                            'AssertionConsumerService:9:Location' => null,
```

Comment:
When adding a new entity. We should not have to add the empty ACS entries.

Reply:
Used the $entity->isManageEntity() for new/existing entities. Also added function to add  empty ACS locations up to the max allowed locations.